### DISCUSSION TITLE
Private members support

### DISCUSSION THREAD

**Discussion by Nice3point** [jeremytammik/RevitLookup#164] [Polls]
The interesting thing is that Solid has material. However, this property is private and there are no public alternatives. Faces of the same solid can have different materials

Do we need to see these private members?

![изображение](https://github.com/jeremytammik/RevitLookup/assets/20504884/9e92d7ef-8847-4c0e-a55e-23aee285909f)

**Comment by ricaun** [comment]
I didn't know that Solid has a private Material, I always use the material in the Face that is public if I remembered right.

Could be interesting to see all private properties but all the time could be annoying to see variable that I'm not supposed to see. 

With a disable/enable configuration and maybe a shortcut to show the private properties temporary in the RevitLookup dialog.

If is a optional feature yes 🙂

**Reply by Nice3point** [reply]
if we add, then only as an option, of course)

**Reply by Nice3point** [reply]
@ricaun https://github.com/jeremytammik/RevitLookup/pull/168

**Comment by jeremytammik** [comment]
yes, agree with ricaun. optional is good, and default to false, for the unaware. how to best ensure that this information can be found and understood by the unaware?
